inc/blocks/ab-test/edit.js

```diff
@@ -191,7 +191,7 @@ const Edit = ( {
 					{ currentPost.type !== 'xb' && (
 						<TextControl
 							label={ <strong>{ __( 'Block Title', 'altis-analytics' ) }</strong> }
-							placeholder={ sprintf( __( '%s (XB %d)', 'altis-analytics' ), currentPost.title, instance.current ) }
+							placeholder={ sprintf( __( '%s %d', 'altis-analytics' ), ( currentPost.title ? currentPost.title : __( 'A/B Test', 'altis-analytics' )  ), instance.current ) }
```

Comment:
You could use the `||` shorthand instead of a ternary here e.g.
```suggestion
							placeholder={ sprintf( __( '%s %d', 'altis-analytics' ), ( currentPost.title || __( 'A/B Test', 'altis-analytics' )  ), instance.current ) }
```

If the thing you are testing is the value you want to use when it's truthy then you can use the `thing || 'default'` syntax to set it. It's just a bit more compact.

Reply:
Thank you, am I also double translating the text with the `__( )` ?

Reply:
Good point, yeah you don't need to translate the string "%s %d" any more.